portability-bootstrap-vm/src/main/java/org/dataportabilityproject/bootstrap/vm/SingleVMMain.java

```diff
@@ -36,10 +40,28 @@ public SingleVMMain(Consumer<Exception> errorCallback) {
 
   public void initializeGateway() {
     ApiMain apiMain = new ApiMain();
-    apiMain.initialize();
-    try {
+
+    try (InputStream stream =
```

Comment:
can this be behind an "environment == localhttps" check? even if you hardcode it to true for now, I'll make it a config option.

Since we normally implement HTTPS thru the cloud project I don't want this code getting into "prod"

Reply:
Better yet can this go somewhere in a local extension's (e.g. portability-cloud-local) initialization code to be extra sure it is not used in "prod"?

Reply:
Just realized this one is within the SingleVm boostrap code so that's OK, but comment still applies to the local HTTPS stuff added to the core code